src/app/components/cds/media-cards/ItemDescription.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames/bind';
+import UnfoldLessIcon from '../../../icons/unfold_less.svg';
+import UnfoldMoreIcon from '../../../icons/unfold_more.svg';
+import FactCheckIcon from '../../../icons/fact_check.svg';
+import styles from './Card.module.css';
+
+const ItemDescription = ({
+  description,
+  title,
+  showCollapseButton,
+  factCheckUrl,
```

Comment:
just more of a note, we will get to it eventually, but there will be the possibility of having other links including/instead of the fact check